go.mod

```diff
@@ -92,4 +92,4 @@ require (
 )
 
 // replace github.com/gophercloud/gophercloud => ./external/gophercloud
-replace github.com/gophercloud/gophercloud => github.com/Wind-River/gophercloud v0.0.0-20231207152242-d06bd3ca4f52
```

Comment:
I think we need to make sure this gets updated in final version of the merge after gopher cloud commit is merged.

Reply:
> I think we need to make sure this gets updated in final version of the merge after gopher cloud commit is merged.

Yes, gophercloud commit is already merged and the new gophercloud version reference is already part of this PR.